workers/unity/Packages/io.improbable.gdk.loadbalancing/Systems/ClientLoadBalancingSystem.cs

```diff
@@ -0,0 +1,64 @@
+#if GDK_PLAYER_LIFECYCLE
```

Comment:
Lifecycle would need to explicitly depend on the load balancing package. Right now, this only depends on generated code that will be present if the lifecycle package.

In the other way around, lifecycle would need to use code defined in this package.

It also more logically belongs here imo

Reply:
I personally think this would set a bad example of how users would have to write / provide their load balancing code.
It's odd to have code for an optional package be inside a required package.

Reply:
I'm really not sure I follow, neither package is required?

Reply:
Discussed offline - going to move to the PL package